frontend/package-lock.json

```diff
@@ -29042,4 +29042,4 @@
       "dev": true
     }
   }
-}
+}
```

Comment:
Came here to check the masterpiece :smile:  Shouldn't there be a newline at the end of file?

Reply:
This is all automatic we don't manually edit stuff there